src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -466,6 +474,14 @@ function registerDiffEditorCommands(): void {
 		primary: undefined,
 		handler: accessor => toggleDiffIgnoreTrimWhitespace(accessor)
 	});
+
+	KeybindingsRegistry.registerCommandAndKeybindingRule({
```

Comment:
@hediet do we want this command to show up in the command palette? Might be nice.

If so, I would expect a MenuRegistry call above this.

Reply:
I think this makes sense.
@Nykseli can you add this?

Reply:
I added it in a new commit. MenuRegistry looked a bit complicated so I hope it's correct